core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/impl/SequencerPartitionContainerImpl.java

```diff
@@ -46,9 +53,15 @@
  * @date 14-May-2012
  * @since 0.1.6
  */
+@Entity
+@Table(name = "SequencerPartitionContainer")
 public class SequencerPartitionContainerImpl extends AbstractSequencerPartitionContainer<SequencerPoolPartition> implements Serializable {
   protected static final Logger log = LoggerFactory.getLogger(SequencerPartitionContainerImpl.class);
 
+  @OneToMany(targetEntity = PartitionImpl.class, cascade = CascadeType.ALL)
+  @JoinTable(name = "SequencerPartitionContainer_Partition", joinColumns = {
+      @JoinColumn(name = "container_containerId") }, inverseJoinColumns = {
+          @JoinColumn(name = "partitions_partitionId") })
   private List<SequencerPoolPartition> partitions = new AutoPopulatingList<SequencerPoolPartition>(PartitionImpl.class);
```

Comment:
Yes, me too. This seems to be in place for the Spring UI. I've left a comment on the [post-hibernatization testing ticket](https://jira.oicr.on.ca/browse/GLT-1387?focusedCommentId=125150&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-125150).